codespell_lib/data/dictionary.txt

```diff
@@ -28324,6 +28397,7 @@ gyph->glyph, gyp,
 gyphs->glyphs, gyps,
 gziniflate->gzinflate
 gziped->gzipped
+göyph->glyph
```

Comment:
That one must be pretty rare :smile:

Reply:
It probably is, though it was found in Emacs. On Swedish qwerty keyboards "l" and "ö" are next to each other, so these typos are not entirely surprising.